pkg/operator/render.go

```diff
@@ -92,6 +92,7 @@ func createDiscoveredControllerConfigSpec(infra *configv1.Infrastructure, networ
 	}
 
 	ccSpec := &mcfgv1.ControllerConfigSpec{
+		NetworkType:         network.Spec.NetworkType,
```

Comment:
By looking at api doc(https://github.com/openshift/api/blob/master/config/v1/types_network.go#L27), it seems like we should be consuming network.status.NetworkType

Reply:
That's true if this wasn't a hack, but `network.Status` only gets filled in after the CNO starts, so that won't work for bootstrap time